how-to-guides/present-cross-validation-results/notebooks/Neptune_cross_validation.ipynb

```diff
@@ -0,0 +1,444 @@
+    "run[\"global/dataset/dataset_size\"] = dataset_size"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "qRaqN0ug1KP_"
+   },
+   "source": [
+    "## Step 3: Log losses and metrics \n",
+    "Training Loop"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "splits = KFold(n_splits=parameters[\"k_folds\"], shuffle=True)\n",
+    "epoch_acc_list, epoch_loss_list = [], []"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "MgGfdruzNB3t"
+   },
+   "outputs": [],
+   "source": [
+    "for fold, (train_ids, _) in enumerate(splits.split(trainset)):\n",
+    "    train_sampler = SubsetRandomSampler(train_ids)\n",
+    "    train_loader = DataLoader(\n",
+    "        trainset, batch_size=parameters[\"bs\"], sampler=train_sampler\n",
+    "    )\n",
+    "    for epoch in range(parameters[\"epochs\"]):\n",
+    "        epoch_acc, epoch_loss = 0, 0.0\n",
+    "        for x, y in train_loader:\n",
+    "            x, y = x.to(device), y.to(device)\n",
+    "            optimizer.zero_grad()\n",
+    "            outputs = model.forward(x)\n",
+    "            _, preds = torch.max(outputs, 1)\n",
+    "            loss = criterion(outputs, y)\n",
+    "            acc = (torch.sum(preds == y.data)) / len(x)\n",
+    "\n",
+    "            # Log batch loss and acc\n",
+    "            run[f\"fold_{fold}/training/batch/loss\"].log(loss)\n",
+    "            run[f\"fold_{fold}/training/batch/acc\"].log(acc)\n",
+    "\n",
+    "            loss.backward()\n",
+    "            optimizer.step()\n",
+    "\n",
+    "        epoch_acc += torch.sum(preds == y.data).item()\n",
+    "        epoch_loss += loss.item() * x.size(0)\n",
+    "\n",
+    "    epoch_acc_list.append((epoch_acc / len(train_loader.sampler)) * 100)\n",
+    "    epoch_loss_list.append(epoch_loss / len(train_loader.sampler))\n",
+    "\n",
+    "    # Log model checkpoint\n",
+    "    torch.save(model.state_dict(), f\"./{parameters['model_name']}\")\n",
+    "    run[f\"fold_{fold}/checkpoint\"].upload(parameters[\"model_name\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "-pC_GC8bXVW2"
+   },
+   "outputs": [],
+   "source": [
+    "# log global acc and loss\n",
+    "run[\"global/metrics/train/mean_acc\"] = mean(epoch_acc_list)\n",
+    "run[\"global/metrics/train/mean_loss\"] = mean(epoch_loss_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "JhE6C9xv-lXK"
+   },
+   "source": [
+    "# Stop run\n",
+    "\n",
+    "<font color=red>**Warning:**</font><br>\n",
+    "Once you are done logging, you should stop tracking the run using the `stop()` method.\n",
+    "This is needed only while logging from a notebook environment. While logging through a script, Neptune automatically stops tracking once the script has completed execution."
```

Comment:
Maybe you can add one more very short section that tell user to go to the link with previously created run and explore results? (You can check our colab template notebook for reference).